CRM/Emailfiling/Hook/BuildForm/AddSettingOutbound.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
+use Civi\Emailfiling\EmailfilingConst;
```

Comment:
This `EmailfilingConst` class should be moved to a service or helper folder following civicrm extension standards. Civicrm uses that format to arrange files rather than namespacing it the ay you have done.